### PR TITLE
fix ethusd of undefined

### PR DESCRIPTION
**Pull request by daiki44**
fix the following error.
```
Cannot read property 'ethusd' of undefined
```
WAK-397

### CODE REVIEW

File: src/store/tokenBalanceList.js [hunk 1]
```diff
@@ -93,8 +93,21 @@ const EtherLatestPriceURL =
...
+      .get(EtherLatestPriceURL)
+      .then(async res => {
+        if (res.data && res.data.result && res.data.result.ethusd) {
+          dispatch(setETHtoUSD(res.data.result.ethusd))
+        } else {
+          await sleep(10 * 1000)
+          dispatch(getETHtoUSD())
+        }
+      })
+      .catch(async e => {
```
Comment (tkmct) [inline src/store/tokenBalanceList.js:106]: I think it's better to display message that says wallet cannot show eth2usd now.

Comment (tkmct) [reply src/store/tokenBalanceList.js:106]: Sorry, I meant dispatch error action and show that it's not available on UI.

Comment (daiki44) [reply src/store/tokenBalanceList.js:106]: updated! I will ask @kiyopikko to implement alert style.
https://github.com/cryptoeconomicslab/gazelle-wallet/blob/bdc3e9429ff3908c5a5634fc38f709dffe96c562/src/components/Base/ErrorAlert.js#L4
スクリーンショット 2020-07-10 10 55 06

Comment (kiyopikko) [reply src/store/tokenBalanceList.js:106]: 🙆‍♂️

**Pull request merged**